src/sass/_mixins.scss

```diff
@@ -42,6 +42,11 @@
   @return map-get($fontSizes, $fontsizeKey);
 }
 
+// $bitSizes one config for font size and line-height 
```

Comment:
I would keep it in bit scss ;)

Reply:
@aju ok, I will move config and function into BIT scss. You have a point, we will keep all the styles related to the component in it's scss.